### ISSUE TITLE
armv7 support

### ISSUE THREAD

**Issue by shajek** [hassio-addons/addon-unifi#369]
Hello,

is there any reason, why is armv7 unsupported ? Unifi controller works OK on XU4, but it would be perfect if it can be done from HA addon.

Thank you

**Comment by sinclairpaul** [comment]
You can see the release notes for https://github.com/hassio-addons/addon-unifi/releases/tag/v1.0.0 as to why 32 bit support was removed.

**Comment by shajek** [comment]
For information to another users, it is posible to run latest version of addon in latest HA on ARMv7 (for me on ODROID XU4).
After cloning git to addon folder and make 3 changes is files all build and work nicely (all components to run it on armv7 exist and work).
The main goal was to use the latest version with 32bit support, but even there I had to modify the dockerfile. So I tried to use the latest version without 32bit support with similar modifications and everything works
